src/main/resources/application.yml

```diff
@@ -12,11 +12,11 @@ spring:
   datasource:
     url: jdbc:postgresql://localhost:5432/postgres?currentSchema=neprozorro
     username: postgres
-    password: 123
+    password: postgres
```

Comment:
avoid changing these parameters, each of us can use their local values without committing them